apps/src/music/views/Callouts.tsx

```diff
@@ -48,6 +52,11 @@ const availableCallouts: AvailableCallouts = {
   },
   'play-sounds-together-block-workspace': {
     selector: `.blocklyWorkspace g[data-id="${BlockTypes.PLAY_SOUNDS_TOGETHER}"] path`,
+    direction: 'left',
+  },
+  'play-sounds-together-block-2-workspace': {
```

Comment:
What does the '2' here refer to? Is this a specific block with the id 'play-sounds-together_2'? Or just the second play together block in the workspace? 

Curious if this means we have to provide a specific block ID in order to target this?

Reply:
`'play-sounds-together-block-2-workspace'` is the callout ID which will point at the block with a levelbuilder-assigned ID of `'play_sounds_together_2'` on the workspace.  

We've had to add IDs to blocks for almost all of the [selectors](https://github.com/code-dot-org/code-dot-org/blob/5a528f3501eae5a08746c3dc7e346d8720a5869f/apps/src/music/views/Callouts.tsx#L21-L73) so far, since we've used CSS selectors to find them onscreen.  Going forward, it might be possible to query Blockly more deeply, but it would require new work.

Reply:
I believe it's the block order in the workspace. So it's the second play together block in the start code. It's good to be able to give students specific directions when they're not sure where to place the blocks. 

@breville how will the moving callout work in an instance where we are guiding students to move a `play sound` block to a `play together` block. Will the arrow rotate to point left?

Reply:
Gotcha - as discussed offline, might be good to eventually have a system where we can provide a block ID directly so we don't need to add a new callout for each specific block? But also understand the desire to keep it limited to specific blocks for now.

As a potential follow-up, would it be useful to store a list of available callout-compatible block IDs that we can reference in levelbuilder?

Reply:
> @breville how will the moving callout work in an instance where we are guiding students to move a play sound block to a play together block. Will the arrow rotate to point left?

For now, it will be an upward-pointing arrow from beginning to end.  This is partly anticipating the possibility that we change icon.  But we could add support for a left-pointing arrow from beginning to end if needed. 

> Gotcha - as discussed offline, might be good to eventually have a system where we can provide a block ID directly so we don't need to add a new callout for each specific block? But also understand the desire to keep it limited to specific blocks for now.

That's a good idea.

> As a potential follow-up, would it be useful to store a list of available callout-compatible block IDs that we can reference in levelbuilder?

That's also a good idea.

Reply:
> as discussed offline, might be good to eventually have a system where we can provide a block ID directly so we don't need to add a new callout for each specific block?

Prepped here: https://github.com/code-dot-org/code-dot-org/pull/61717